workspace.json

```diff
@@ -1186,6 +1186,17 @@
             "passWithNoTests": true
           }
         },
+        "extract-strings": {
```

Comment:
Do we need to call these targets during the build process?

Reply:
Not atm, this is done manually

@magnearun correct me if I'm wrong

Reply:
This is manual atm, not sure when in the process we want to do this. This extracts the strings from the code and uploads them into Contentful